packages/headless/src/utils/validate-payload.ts

```diff
@@ -13,6 +13,11 @@ export const requiredNonEmptyString = new StringValue({
   emptyAllowed: false,
 });
 
+export const nonEmptyString = new StringValue({
```

Comment:
This definition was used often, so I replaced with this everywhere I could see

Reply:
Just a thought: Is there a possibility that there's some stateful properties in a`StringValue` instance from bueno ?

Should it be a function that returns a `new StringValue({..})` instead ?

Reply:
I don't think it's a possibility since it's a definition and shouldn't be modified, we also have had requiredNonEmptyString / requiredEmptyAllowedString for the longest time